sklearn/linear_model/_ridge.py

```diff
@@ -1502,7 +1502,10 @@ def fit(self, X, y, sample_weight=None):
                 alpha_score = scorer(
                     _IdentityEstimator(), predictions.ravel(), y.ravel())
                 if self.store_cv_values:
-                    self.cv_values_[:, i] = predictions.ravel()
+                    # compute the predictions in the original space
+                    self.cv_values_[:, i] = (
+                        predictions.ravel() / np.tile(sqrt_sw, n_y)
+                        + np.repeat(y_offset, X.shape[0]))
```

Comment:
Is it good to use ``np.repeat(y_offset, X.shape[0])``? Seems that it's not memory efficient?
I'm still unable to understand why your version is better than mine.
```
    if y.ndim == 2:    
        y_true = y / sqrt_sw[:, np.newaxis] + y_offset
        y_pred = predictions / sqrt_sw[:, np.newaxis] + y_offset
    else:
        y_true = y / sqrt_sw + y_offset
        y_pred = predictions / sqrt_sw + y_offset
```